samples/EventImporter/Provider.Tests/AuthorizationTokenReplacementMiddleware.cs

```diff
@@ -0,0 +1,29 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace Provider.Tests
+{
+    public class AuthorizationTokenReplacementMiddleware
```

Comment:
I'd probably remove all the auth stuff from this example. It just adds noise whereas the other example already shows how to do auth.